src/components/Card/Card.js

```diff
@@ -6,25 +9,66 @@ import "./__text/card__text.css";
 import "./__title/card__title.css";
 import "./__origin/card__origin.css";
 
-import image from "../../images/image_08.png";
+import "./__like-flag/card__like-flag.css";
+import "./__like-flag/_active/card__like-flag_active.css";
+import "./__delete-trash/card__delete-trash.css";
+import "./__category/card__category.css";
+import "./__button-description/card__button-description.css";
+import "./__button-description/_non-active/card__button-description_non-active.css";
+import MainApi from "../../utils/MainApi";
+
+function Card({
+  card,
+  isLoggedIn,
+  keyword,
+  updateSavedArticles,
+  savedArticles,
+}) {
+  const [isShown, setIsShown] = useState(false);
+  const [isLiked, setIsLiked] = useState(false);
+  const isAlreadySaved = savedArticles.some((c) => c.title === card.title);
 
-function Card({ children }) {
   return (
     <li className="card">
-      <img className="card__picture" src={image} alt={`A Card name`} />
+      <img className="card__picture" src={`${card.urlToImage}`} alt={""} />
```

Comment:
`alt={""}`

Every tag `img` should have an informative attribute `alt` to show the `alternative` text if the image is not uploaded and it’s needed for `screen readers`